geiger.go

```diff
@@ -10,17 +11,22 @@ import (
 )
 
 const sr = beep.SampleRate(44100)
-const memSampleRate = 100 * time.Millisecond
+const memSampleRate = 500 * time.Millisecond
+const baseline = 160
```

Comment:
ideally we could measure this empirically somehow, but I suppose this is better than nothing

Reply:
Yeah, it was a bit "twitchy" with 100... At least now if you make a burst of allocations you will feel the pain for half a second :D